besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java

```diff
@@ -417,14 +417,18 @@ public String build() {
   private void detectJemalloc(final List<String> lines) {
     Optional.ofNullable(Objects.isNull(environment) ? null : environment.get("BESU_USING_JEMALLOC"))
```

Comment:
I'm sorry but I don't understand this one, BESU_USING_JEMALLOC is still in the script

if [ "\$BESU_USING_JEMALLOC" != "FALSE" -a "\$BESU_USING_JEMALLOC" != "false" ]; then

and may or not exist as a env var, actually that situation is expected in the current code

            () -> {
              // in case the user is using jemalloc without BESU_USING_JEMALLOC env var
              try {
                final String version = PlatformDetector.getJemalloc();
                lines.add("jemalloc: " + version);
              } catch (final Throwable throwable) {
                logger.info(
                    "jemalloc library not found, memory usage may be reduced by installing it");
              }
            });

So now the detecJemalloc outputs one of 3 warnings:

"besu_using_jemalloc is present but is not set to true, jemalloc library not loaded"
"besu_using_jemalloc is present but we failed to load jemalloc library to get the version"
"jemalloc library not found, memory usage may be reduced by installing it"

Actually, with this version now if no action is taken by the clients (meaning they do not create a BESU_USING_JEMALLOC enc var, like running it out of the box) the code that actually runs is that one above that exixts already, that will load jemalloc and outputs the version or it fails to load jemalloc and outputs that "jemalloc library not found, memory usage may be reduced by installing it".

Let me know if this makes sense...

Cheers.